www/index.md

```diff
@@ -0,0 +1,26 @@
+# Compensated
+Build products. Get paid.
+
+
+### Expand Your Sales Channels
+You started with a website, and now you have an iOS app.
+And oh yea, there’s that old Gumroad storefront you were experimenting with that has some recurring payments set up.
+
+No matter how you get (or got!) paid, Compensated standardizes the different payment processers data structures and event types so you can add sales channels without losing momentum on your customer facing features.
```

Comment:
processors'

Edit: 
Actually... I want to comment on more than just the spelling and missing possessive. I like the message but it feels a bit wordy. How about?:

> No matter how you get paid, Compensated standardizes the different payment processors' data structures and event types. So you can make money while focusing on your product, rather than on obscure payment details.

Reply:
I dig it!